src/components/TransactionListOptions/TransactionListOptionsTs.ts

```diff
@@ -110,34 +133,7 @@ export class TransactionListOptionsTs extends Vue {
    * @type {{publicKey: string, label: string}[]}
    */
   protected get signers(): {publicKey: string, label: string}[] {
-    if (!this.currentWallet) return []
-
-    const self = [
-      {
-        publicKey: this.currentWallet.values.get('publicKey'),
-        label: this.currentWallet.values.get('name'),
-      },
-    ]
-
-    const multisigInfo = this.currentWalletMultisigInfo
-    if (!multisigInfo) return self
-
-    // in case "self" is a multi-signature account
-    if (multisigInfo && multisigInfo.isMultisig()) {
-      self[0].label = self[0].label + this.$t('label_postfix_multisig')
-    }
-
-    // add multisig accounts of which "self" is a cosignatory
-    if (multisigInfo) {
-      const service = new WalletService(this.$store)
-      return self.concat(...multisigInfo.multisigAccounts.map(
-        ({publicKey}) => ({
-          publicKey,
-          label: service.getWalletLabel(publicKey, this.networkType) + this.$t('label_postfix_multisig'),
-        })))
-    }
-
-    return self
+    return new MultisigService(this.$store).getSigners(`${this.$t('label_postfix_multisig')}`)
```

Comment:
remove text paremeter, getSigners can use the translation module

Reply:
passed i18n to an optional argument of multisigService constructor